galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
https://wiki.galaxyproject.org/Admin/Tools/ToolConfigSyntax#A.3Cparam.3E_tag_set under type="data" you can find documentation.